### ISSUE TITLE
dataset "GAN-G.csv", "GAN-D.csv" and IDS model "Keras_IDS_CNN.h5"

### ISSUE THREAD

**Issue by 89935** [thetinybug/thesis-IDSGAN#2]
Hi, I am trying to reproduce your code in "https://github.com/thetinybug/thesis-IDSGAN/blob/dev-checkingWGAN/IDS_WGAN_and_GenerateAdversarialTraffic_(Keras)-Loss(-).ipynb". Could you tell me where I can find the preprocessed dataset "GAN-G.csv", "GAN-D.csv" and the trained IDS model "Keras_IDS_CNN.h5" ? Looking foward to your reply! If possible, could you update your data preprocessing code and IDS model training code or tell me the procedure. Thanks a lot!
